spec/lib/insights/api/common/custom_exceptions_spec.rb

```diff
@@ -10,5 +11,15 @@
         )
       end
     end
+
+    context "when the query is Symbol" do
+      let(:exception) { double(:class => "Pundit::NotAuthorizedError", :query => :create?, :record => record) }
```

Comment:
Small nitpick, but what do you think about moving this below line 3, changing to use `:query => query` and then use `let(:query) { :create? }` for this context and `let(:query) { "create?" }` for the other context?